### DISCUSSION TITLE
Shared VPC and Subnet sharing with projects ?

### DISCUSSION THREAD

**Discussion by HarshalSanap007** [GoogleCloudPlatform/cloud-foundation-fabric#2759] [Q&A] [answered]
Hi All, 
I've been trying this Fast-fabric and I'm halfway through it, so while deploying the simple networking or networking separate envs , the VPCs get created in their projects. Still, the subnets that are part of these VPCs are not shared with the projects created in project factory. Is there any module that needs to be enabled so that these subnets are shared with the teams project ?

Thanks

**Comment by ludoo** [comment]
What we typically do is grant network user role on the VPC projects to the right principals, so subnets can be consumed from service projects. If you need to restrict access so that specific projects can only consume certain subnets, use the relevant organization policy in the service project. We have an example somewhere of doing it...

**Reply by HarshalSanap007** [reply]
Hey ludoo, 
thanks for real quick response, continuing to that, I will be specific in 2-networking-a-simple, it creates a hub and spoke model and 3 projects and 3 vpc respectively inside them, I don't see "compute network user" given to any service account or user of that project, the important thing is that I don't see any projects attached to the dev/prod host projects.
Am I missing anything? FYI- I created the application project first from the project factory and then deployed the 2-networking-a-simple.

**Reply by HarshalSanap007** [reply]
do we need to explicitly create .tfvars for this so that we can add the value for variables "vpc_host_projects" ?

**Reply by HarshalSanap007** [reply, marked as answer]
Hi @ludoo , thanks for your help , I found the issue.
I had commented the shared_vpc_service_config block 

shared_vpc_service_config:
  host_project: aus-prod-net-spoke-1
  network_users:
    - gcp-devops
    
   Thanks for your time
